SwiftEvolve/Package.swift

```diff
@@ -59,7 +59,7 @@ if getenv("SWIFTCI_USE_LOCAL_DEPS") == nil {
     // Building standalone.
     package.dependencies += [
         .package(url: "https://github.com/apple/swift-tools-support-core.git", .branch("main")),
-        .package(url: "https://github.com/apple/swift-syntax.git", .branch("main")),
+        .package(path: "../../swift-syntax")
```

Comment:
I think this is just a leftover from your local testing setup, right?

Reply:
Whoops. 😅